app/uk/gov/hmrc/gvmshaulierstatus/services/HaulierStatusService.scala

```diff
@@ -54,8 +54,9 @@ class HaulierStatusService @Inject()(haulierStatusRepository: HaulierStatusRepos
       case Nil =>
         logger.info("Setting haulier status to AVAILABLE")
         customsServiceStatusConnector.updateStatus(appConfig.haulierServiceId, AVAILABLE)
-      case _ =>
-        logger.info("Setting haulier status to UNAVAILABLE")
+      case correlationIds =>
+        logger.warn("Setting haulier status to UNAVAILABLE")
+        logger.info(s"${correlationIds.length} Correlation ids found (curtailed): ${correlationIds.takeRight(10).mkString(", ")}")
```

Comment:
can we do this size at the db level rather than in the code ? 
who knows there might be 1000s of such ids or just a handful... if there are too many we don't need to load them all..
i remember there is `limit` option on the mongo side

Reply:
Also can you add logging in the FE here https://github.com/hmrc/customs-service-status-frontend/blob/main/app/uk/gov/hmrc/customsservicestatusfrontend/controllers/DashboardController.scala#L42 
you can do a WARN log to see which component is UNAVAILBALE ?

Reply:
would it make sense to add the limit to `haulierStatusRepository.findAllOlderThan` perhaps?

Reply:
yeah, you can use .limit(10) after .find() inside the query... 
but not sure if we agreed to log the whole count in the service, if you dont need to fix anything there...